openshift-kube-apiserver/admission/customresourcevalidation/apiserver/validate_apiserver.go

```diff
@@ -200,7 +200,9 @@ func validateTLSSecurityProfileType(fieldPath *field.Path, profile *configv1.TLS
 			errs = append(errs, field.Required(fieldPath.Child("intermediate"), fmt.Sprintf(typeProfileMismatchFmt, profile.Type)))
 		}
 	case configv1.TLSProfileModernType:
```

Comment:
we could add the new profile to `availableTypes`

Reply:
Luis is on PTO, you ok with us adding this in a followup PR if this moves forward before he's back?